<launch> 
    <node name="analytics" pkg="analytics" type="analytics" output="screen" />

    <param name="connect" value="true" type="bool" />

    <param name="db_url" value="https://my-project-1516369881504.firebaseio.com/RelCos/" />

    <param name="session" value="25" type="int" />

</launch>